workout_portal/urls.py

```diff
@@ -1,10 +1,10 @@
 from django.urls import path
 
-# from . import views
+from . import views
 
 
 urlpatterns = [
-    # path('', controller.index, name='index'),
-    # path('training/add', controller.add_training, name='add_training'),
-    # path('training/<int:training_id>/', controller.get_training, name='get_training'),
+    path('', controller.index, name='index'),
```

Comment:
why you uncommented this code? This endpoints already defined in `django_learning/urls.py` file with ViewSets